contracts/RefunderFactory.sol

```diff
@@ -0,0 +1,40 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.4;
+
+import "./IRegistry.sol";
+import "./IRefunder.sol";
+import {Clones} from "@openzeppelin/contracts/proxy/Clones.sol";
+
+contract RefunderFactory {
+    address _registry;
```

Comment:
I think that this should be public variable and have automatically created getter + add unit test as-well